sonic_platform_base/module_base.py

```diff
@@ -35,6 +35,9 @@ class ModuleBase(device_base.DeviceBase):
     #Module state is Online when fully operational
```

Comment:
Consider a use-case: 
Temperature sensor on a LC or FC crosses its critical threshold. Platform expected to power-down/shutdown the card. thermCtlD (via policy file) asks platform to do the needful. 

Per the above card state definitions, card can't be moved to MODULE_STATUS_OFFLINE as its not user/admin-config initiated. Card to be moved to MODULE_STATUS_FAULT. However, card is actually powered-off by the platform (to protect the board/card) whereas MODULE_STATUS_FAULT definition says card is still powered-on though in a fault state. 

Propose to resolve this ambiguity by introducing MODULE_STATUS_POWERED_OFF (or MODULE_STATUS_POWERED_DOWN).

Reply:
Added MODULE_STATUS_POWERED_DOWN.